src/main/scala/interplay/PlaySonatypeBase.scala

```diff
@@ -13,6 +14,7 @@ object PlaySonatypeBase extends AutoPlugin {
   override def requires = Sonatype
 
   override def projectSettings = Seq(
+    playBuildPromoteSonatype in ThisBuild := true,
```

Comment:
Doesn't this interfere with https://github.com/playframework/interplay/blob/master/src/main/scala/interplay/PlayReleaseBase.scala#L51-L54 ?

Reply:
Yes, this is how the Sonatype releasing selected.

Reply:
I think adding that is causing the error in the scripted test:

```
[info] [error] [MISSING_CREDENTIAL] No credential is found for oss.sonatype.org. Prepare ~/.sbt/(sbt_version)/sonatype.sbt file.
```

Reply:
I can no longer reproduce. 

I don't know what's going on.